### ISSUE TITLE
Latitude and longitude data incorrect for bulk loaded records

### ISSUE THREAD

**Issue by szaborac** [ArctosDB/arctos#5137]
**Describe the bug**
I am bulk loading catalog records into Arctos via a CSV file. Arctos is overriding the information I am providing regarding latitude and longitude - and sometimes getting it very, very wrong.

**To Reproduce**
1. Search for Guid UAMb:herb:61254
2. Note the latitude and longitude values assigned by Arctos: 49.766667, -94.55
3. See the attached CSV example, where the latitude is 66.17 and the longitude is -94.42. 4. 
4. Look in Locality tab for the Georeference Source and the Georeference Protocol 

**Expected behavior**
Arctos should have located the specimen in Northwest Territories - not Ontario - as I specified in the bulk loaded CSV file. 

Note: Arctos has re-assigned the latitude and longitude of every record I have bulk loaded today (11 records). 

**Data**
See attached
[Generic bulkloader data entry - upload 3.csv](https://github.com/ArctosDB/arctos/files/9728405/Generic.bulkloader.data.entry.-.upload.3.csv)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 10
 - Browser [e.g. chrome, safari] Safari

**Priority** 
Priority 1.

**Comment by szaborac** [comment]
I don't know why it converted the word "herb:" after the UAMb: into a leaf. ?

**Comment by dustymc** [comment]
Bah, a script was being dumb. I've shut it down and I'll undo what it's done.

**Comment by dustymc** [comment]
All coordinate changes made by the bot should now be rolled back.

**Comment by sharpphyl** [comment]
@dustymc   This is slightly different from the above concern about the bot changing coordinates.

Sometimes, I don't want the locality to be geolocated.  For the record https://arctos.database.museum/guid/DMNS:Inv:8702, we have a locality but low confidence in our interpretation of what it means.  Therefore, I did not want to geolocate the specimen even though I entered the data with a remark describing the  insufficiency of the data.   

Is there a way to choose whether we want the bot coordinates based on the locality text?
